checker/tests/lock/FlowExpressions.java

```diff
@@ -24,17 +24,5 @@ public void method() {
         synchronized(getm()) {
             getm().field = new Object();
         }
-
```

Comment:
It was testing a situation where "itself" could refer to either a variable (within the block) or "<self>" (outside of this block).